scheduler/ecs/ecs.go

```diff
@@ -527,16 +527,18 @@ func (m *Scheduler) loadBalancer(ctx context.Context, app *scheduler.App, p *sch
 	return l, nil
 }
 
-func (m *Scheduler) removeLoadBalancer(ctx context.Context, app string, p string) error {
-	l, err := m.findLoadBalancer(ctx, app, p)
+func (m *Scheduler) removeLoadBalancers(ctx context.Context, app string) error {
```

Comment:
Was this called somewhere else before?  I'm not sure how this stops the previous behavior - only that it adds the behavior that it will now make sure that the LBs are destroyed when `Remove` is called?